miner/worker.go

```diff
@@ -1001,6 +1001,35 @@ func (w *worker) commitTransactions(env *environment, txs *types.TransactionsByP
 		//
 		// We use the eip155 signer regardless of the current hf.
 		from, _ := types.Sender(env.signer, tx)
+
+		// not prioritising conditional transaction, yet.
+		if options := tx.GetOptions(); options != nil {
+			if options.BlockNumberMax != big.NewInt(0) {
```

Comment:
Could we use https://pkg.go.dev/math/big#Int.Cmp rather then equality with ref type?

Reply:
Done, thanks.